airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -0,0 +1,110 @@
+    :param connections: AWS Glue connections to be used by the job.
+    :type list
+    :param retry_limit: The maximum number of times to retry this job if it fails
+    :type int
+    :param num_of_dpus: Number of AWS Glue DPUs to allocate to this Job.
+    :type int
+    :param region_name: aws region name (example: us-east-1)
+    :type region_name: str
+    :param s3_bucket: S3 bucket where logs and local etl script will be uploaded
+    :type str
+    :param iam_role_name: AWS IAM Role for Glue Job Execution
+    :type str
+    """
+    template_fields = ()
+    template_ext = ()
+    ui_color = '#ededed'
+
+    @apply_defaults
+    def __init__(self,
+                 job_name='aws_glue_default_job',
+                 job_desc='AWS Glue Job with Airflow',
+                 script_location=None,
+                 concurrent_run_limit=None,
+                 script_args={},
+                 connections=[],
+                 retry_limit=None,
+                 num_of_dpus=6,
+                 aws_conn_id='aws_default',
+                 region_name=None,
+                 s3_bucket=None,
+                 iam_role_name=None,
+                 *args, **kwargs
+                 ):
+        super(AWSGlueJobOperator, self).__init__(*args, **kwargs)
+        self.job_name = job_name
+        self.job_desc = job_desc
+        self.script_location = script_location
+        self.concurrent_run_limit = concurrent_run_limit
+        self.script_args = script_args
+        self.connections = connections
+        self.retry_limit = retry_limit
+        self.num_of_dpus = num_of_dpus
+        self.aws_conn_id = aws_conn_id,
+        self.region_name = region_name
+        self.s3_bucket = s3_bucket
+        self.iam_role_name = iam_role_name
+
+    def execute(self, context):
+        """
+        Executes AWS Glue Job from Airflow
+        :return:
+        """
+        task_instance = context['ti']
+        glue_job = AwsGlueJobHook(job_name=self.job_name,
+                                  desc=self.job_desc,
+                                  concurrent_run_limit=self.concurrent_run_limit,
+                                  script_location=self.script_location,
+                                  conns=self.connections,
+                                  retry_limit=self.retry_limit,
+                                  num_of_dpus=self.num_of_dpus,
+                                  aws_conn_id=self.aws_conn_id,
+                                  region_name=self.region_name,
+                                  s3_bucket=self.s3_bucket,
+                                  iam_role_name=self.iam_role_name)
+
+        self.log.info("Initializing AWS Glue Job: %s",self.job_name)
+        glue_job_run = glue_job.initialize_job(self.script_args)
+        task_instance.xcom_push(key='run_id', value=glue_job_run['JobRunId'])
+        self.log.info("AWS Glue Job: %s status: %s. Run Id: %s",glue_job_run['JobRunId'],self.job_name,glue_job_run['JobRunState'])
```

Comment:
```suggestion
        :return: the id of the current glue job.
        """
        glue_job = AwsGlueJobHook(job_name=self.job_name,
                                  desc=self.job_desc,
                                  concurrent_run_limit=self.concurrent_run_limit,
                                  script_location=self.script_location,
                                  conns=self.connections,
                                  retry_limit=self.retry_limit,
                                  num_of_dpus=self.num_of_dpus,
                                  aws_conn_id=self.aws_conn_id,
                                  region_name=self.region_name,
                                  s3_bucket=self.s3_bucket,
                                  iam_role_name=self.iam_role_name)

        self.log.info("Initializing AWS Glue Job: %s",self.job_name)
        glue_job_run = glue_job.initialize_job(self.script_args)
        self.log.info("AWS Glue Job: %s status: %s. Run Id: %s",glue_job_run['JobRunId'],self.job_name,glue_job_run['JobRunState'])
        return glue_job_run['JobRunId']
```

Note that you can just `return` in the execute and the value will be pushed to xcom which you can then get via the key `return_value`.